File: java/moduleB/src/main/java/dev/potaro/multiprojectexample/b/MultiProjectExampleB.java
package dev.potaro.multiprojectexample.b;

import org.bukkit.plugin.java.JavaPlugin;

public final class MultiProjectExampleB extends JavaPlugin {

    @Override
    public void onEnable() {
        getLogger().info("Enabled test plugin B");
    }

    @Override
    public void onDisable() {
        getLogger().info("Disabled test plugin B");
    }
}

File: kotlin/moduleB/src/main/kotlin/dev/potaro/multiprojectexample/b/MultiProjectExampleB.kt
package dev.potaro.multiprojectexample.b

import org.bukkit.plugin.java.JavaPlugin

class MultiProjectExampleB : JavaPlugin() {

    override fun onEnable() {
        logger.info("Enabled test plugin B")
    }

    override fun onDisable() {
        logger.info("Disabled test plugin B")
    }
}
